test/integration/verify/controls/google_compute_project_info.rb

```diff
@@ -1,14 +1,27 @@
-title 'Test GCP Compute Project Info'
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file in README.md and
+#     CONTRIBUTING.md located at the root of this package.
+#
+# ----------------------------------------------------------------------------
 
-gcp_project_id = attribute(:gcp_project_id, default: '', description: 'The GCP project identifier.')
-
-control 'gcp-compute-project-info-1.0' do
+title 'Test GCP google_compute_project_info resource.'
 
+gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
+control 'google_compute_project_info-1.0' do
   impact 1.0
-  title 'Ensure GCP Compute Project Info has the correct properties.'
+  title 'google_compute_project_info resource test'
 
   describe google_compute_project_info(project: gcp_project_id) do
-    it { should exist }
-    its('name') { should eq gcp_project_id }
+  	it { should exist }
+  	its('default_service_account') { should match "developer.gserviceaccount.com" }
+  	it { should_not be_has_enabled_oslogin }
```

Comment:
This could be updated to e.g. 
```
it { should_not have_enabled_oslogin }
```
but given that both can be possible - can we remove this line altogether to avoid having to make a manual change outside of terraform?